setup.py

```diff
@@ -2003,11 +2003,11 @@ def detect_ctypes(self, inc_dirs, lib_dirs):
             ffi_inc = find_file('ffi.h', [], inc_dirs)
         if ffi_inc is not None:
             ffi_h = ffi_inc[0] + '/ffi.h'
+            # Just check that the file is not empty
```

Comment:
Based on my "git blame"-fu it was introduced by this commit which is a large merge commit from a non-existent branch in the git repo:

commit 49fd7fa4431da299196d74087df4a04f99f9c46f
Author: Thomas Wouters <thomas@python.org>
Date:   Fri Apr 21 10:40:58 2006 +0000

    Merge p3yk branch with the trunk up to revision 45595. This breaks a fair
    number of tests, all because of the codecs/_multibytecodecs issue described
    here (it's not a Py3K issue, just something Py3K discovers):
    http://mail.python.org/pipermail/python-dev/2006-April/064051.html